smtp.go

```diff
@@ -47,11 +46,22 @@ func (m *SMTPEmailBackend) Init(conf map[string]string) error {
 
 	host, port, err := net.SplitHostPort(conf["SMTPAddress"])
 	if err != nil {
-		return err
-	}
-	if port == "" {
-		port = conf["SMTPPort"]
+		if e, ok := err.(*net.AddrError); ok && e.Err == "missing port in address" {
```

Comment:
I think we should add a comment here to explain - "in case there was an error getting the port we try to get it from a dedicated variable"

Reply:
791f66e resolves

Reply:
good catch